app/bundles/CoreBundle/Doctrine/Connection/ConnectionMasterSlaveWrapper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Doctrine\Connection;
+
+use Doctrine\DBAL\Connections\MasterSlaveConnection;
+use Doctrine\DBAL\Exception;
+
+class ConnectionMasterSlaveWrapper extends MasterSlaveConnection
```

Comment:
We should use `PrimaryReadReplicaConnection` because `MasterSlaveConnection` is deprecated. As far as I see it's only a name adjustment here and in config, but it will be removed in future releases.

Reply:
@patrykgruszka I already implemented this, thank you for the notice